addons/website/static/src/js/website.backendDashboard.js

```diff
@@ -140,6 +139,15 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
 
             var tick_values = self.getPrunedTickValues(chart_values[0].values, 5);
 
+            chart.interactiveLayer.tooltip.contentGenerator(function(d) {
+                var date = moment(d.value);
+                return  '<table><thead><tr><td colspan="3"><strong>'+ d.series[0].key +'</strong></td></tr></thead>'+
```

Comment:
You should avoid that. Why did you did it ? If necessary embed it in a template somewhere.